.vale.ini

```diff
@@ -0,0 +1,9 @@
+StylesPath = .github/vale-styles
+MinAlertLevel = suggestion
+
+[formats]
+mdx = md
+
+[*.md]
+BasedOnStyles = messaging
+TokenIgnores = "^\s*\(!.*\!\)\s*$","[\{\}]"
```

Comment:
TODO: I need to add a comment about what this ignores (our include syntax and partial parameters) and make sure the token ignores work as expected.